test/data/housing.txt

```diff
@@ -0,0 +1,506 @@
+24.00	0.00632	18.00	2.310	0	0.5380	6.5750	65.20	4.0900	1	296.0	15.30	396.90	4.98
+21.60	0.02731	0.00	7.070	0	0.4690	6.4210	78.90	4.9671	2	242.0	17.80	396.90	9.14
```

Comment:
Last time i asked about this dataset, I believe we couldn't use it. 
I switched my test to the wine quality. 
Might have to use the regression dataset that is in the repo already. 

 @shauheen @artidoro to comment further

Reply:
From all I could find online, this dataset appears to be in the public domain.

This site claims the dataset is in the public domain:
http://b-course.hiit.fi/obc/housingexpl.html
“ B-Course provides a public domain data set called Boston Housing Data so that one can try out B-Course without own data. Below you can find some details of our example data set.”

The dataset is also distributed in an R package here:
https://www.rdocumentation.org/packages/mlbench/versions/2.1-1

It's used in books as well: https://github.com/rasbt/python-machine-learning-book-2nd-edition/blob/master/code/ch10/housing.data.txt

I've added the attribution to the README.

By the way, this is a different dataset from the "King County Housing" (kc_housing) dataset.

Reply:
CELA has approved the re-distribution of the Boston Housing dataset.